packages/@react-types/dialog/src/index.d.ts

```diff
@@ -45,15 +45,25 @@ export interface SpectrumDialogProps extends DOMProps, StyleProps {
 }
 
 export interface SpectrumAlertDialogProps extends DOMProps, StyleProps {
+  /** The [visual style](https://spectrum.adobe.com/page/dialog/#Options) of the AlertDialog.  */
   variant?: 'confirmation' | 'information' | 'destructive' | 'error' | 'warning'
+  /** The title of the AlertDialog. */
   title: string,
+  /** The contents of the AlertDialog. */
   children: ReactNode,
+  /** The label to display within the cancel button. */
   cancelLabel?: string,
+  /** The label to display within the confirm button. */
   primaryLabel?: string,
+  /** The label to display within the secondary button. */
   secondaryLabel?: string,
+  /** Whether the confrim button is disabled. */
   isConfirmDisabled?: boolean,
```

Comment:
also we don't call this confirm button anymore so we should update this name. `isPrimaryActionDisabled`?
Should we disable both with one prop or one prop per action button?

Reply:
I think we should have one prop per action button since there are probably scenarios where only one should be disabled. e.g. "Publish" (aka primary) is disabled due to some invalid input/closed publish window or something, but a "Save as draft and exit"/"Schedule auto publish" would be clickable still